www/front_src/src/helpers/dynamicImport.js

```diff
@@ -64,9 +62,9 @@ export const dynamicImport = (basename, parameters) =>
 
     try {
       // dynamically import css if external component needs one
-      if (css && css.length > 0) {
-        await systemCss.fetch({ address: basename + css });
-      }
+      // if (css && css.length > 0) {
```

Comment:
We can get rid of this

Reply:
We will get rid of this with the Module Federation PR